src/Carbon/Carbon.php

```diff
@@ -1650,6 +1650,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
+     *
+     * @return bool
+     */
+    public function isNextWeek()
+    {
+        return $this->weekOfYear === static::now($this->getTimezone())->addWeek()->weekOfYear;
+    }
+
+    /**
+     * Determines if the instance is within the last week
+     *
+     * @return bool
+     */
+    public function isLastWeek()
+    {
+        return $this->weekOfYear === static::now($this->getTimezone())->subWeek()->weekOfYear;
+    }
+
+    /**
+     * Determines if the instance is within the next month
+     *
+     * @return bool
+     */
+    public function isNextMonth()
+    {
+        return $this->month === static::now($this->getTimezone())->addMonth()->month;
```

Comment:
Unfortunatelly this is not always working:
```php
Carbon::setTestNow(Carbon::create(2016, 1, 31));
$this->assertTrue(Carbon::create(2016, 2, 27)->isNextMonth());
```

Months comparison should use `addMonthsNoOverflow` and `subMonthsNoOverflow`.